### ISSUE TITLE
Route [] not defined.

### ISSUE THREAD

**Issue by kieran-s** [kanalumaddela/laravel-steam-login#48]
I'm probably missing something basic... but anyway...
I've also done `php artisan cache:clear` as issue #46 had and it has not changed my error.

Laravel 5.8
PHP 7.3

As per readme.md, required package, did force vendor publish, added routes w/ uses, created controller and pasted the controller on this page (https://github.com/kanalumaddela/laravel-steam-login/wiki/2.x-Usage-Laravel) into my controller

Navigate to /login/steam and I get the error.
`InvalidArgumentException Route [] not defined.`

Stack trace:
```
57
InvalidArgumentException 
…/vendor/laravel/framework/src/Illuminate/Routing/UrlGenerator.php388
56
Illuminate\Routing\UrlGenerator route
…/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php782
55
 route
…/vendor/kanalumaddela/laravel-steam-login/src/SteamLogin.php204
54
kanalumaddela\LaravelSteamLogin\SteamLogin buildLoginUrl
…/vendor/kanalumaddela/laravel-steam-login/src/SteamLogin.php427
53
kanalumaddela\LaravelSteamLogin\SteamLogin getLoginUrl
…/vendor/kanalumaddela/laravel-steam-login/src/SteamLogin.php405
52
kanalumaddela\LaravelSteamLogin\SteamLogin redirectToSteam
…/vendor/kanalumaddela/laravel-steam-login/src/Http/Controllers/AbstractSteamLoginController.php65
51
kanalumaddela\LaravelSteamLogin\Http\Controllers\AbstractSteamLoginController redirectToSteam
…/vendor/kanalumaddela/laravel-steam-login/src/Http/Controllers/AbstractSteamLoginController.php57
50
kanalumaddela\LaravelSteamLogin\Http\Controllers\AbstractSteamLoginController login
…/vendor/laravel/framework/src/Illuminate/Routing/Controller.php54
```

My controller if interested (the diedump/DD is because I don't want it to save a user just yet, I want it to hit the auth section so I can see it's working)
```
<?php

namespace App\Http\Controllers\Auth;

use Illuminate\Http\Request;
use Illuminate\Support\Facades\Auth;
use Illuminate\Support\Facades\Hash;
use kanalumaddela\LaravelSteamLogin\Http\Controllers\AbstractSteamLoginController;
use kanalumaddela\LaravelSteamLogin\SteamUser;
use App\User;

class SteamLoginController extends AbstractSteamLoginController
{
    /**
     * {@inheritdoc}
     */
    public function authenticated(Request $request, SteamUser $steamUser)
    {
        dd('welcome '.$steamUser->name);
        // find user by their steam account id, example assumes `steam_account_id` on `users` table
        $user = User::where('steam_account_id', $steamUser->accountId)->first();

        // if the user doesn't exist, create them
        if (!$user) {
            $steamUser->getUserInfo(); // retrieve and set user info pulled from steam

            $user = User::create([
                'name' => $steamUser->name, // personaname
                'steam_account_id' => $steamUser->accountId,
            ]);
        }

        // login the user using the Auth facade
        Auth::login($user);

        // let the extended controller handle redirect back to the page the user was just on
    }
}
```

**Comment by kanalumaddela** [comment]
Late response, but if I had to take a guess, there seems to be an issue when using the `route()` helper func when generating URLs at https://github.com/kanalumaddela/laravel-steam-login/blob/781348c4a3052b42e5de409877af77bb4dda73e0/src/SteamLogin.php#L204

It uses the value from the config, but it's not returning anything. Can you make sure the config file exists in your project and matches the current one in the repo? https://github.com/kanalumaddela/laravel-steam-login/blob/master/config/steam-login.php.
A side note: you shouldn't be clearing any form of cache because you shouldn't be caching while in development.

**Comment by kieran-s** [comment]
I figured as much re: cache, but hey, I figure when other things fail, it's worth a shot :P

Is that config file supposed to be in `(project)/config`?
If so, it looks like it does not exist, so that is probably the problem

This time I ran `php artisan vendor:publish` then selected the number that referenced this project and that seemed to create the steam-login config successfully.
Looks like last time when I ran `php artisan vendor:publish --force --provider kanalumaddela\LaravelSteamLogin\SteamLoginServiceProvider` it did not properly copy the file across.

I wasn't sure if a missing file in `/config` was the issue, but then I've never used the vendor:publish command with --force and --provider flags, so I just assumed it was doing something a little different.

It is all working now. Thanks!
